vmdb/app/models/customization_script.rb

```diff
@@ -1,4 +1,7 @@
 class CustomizationScript < ActiveRecord::Base
   include NewWithTypeStiMixin
   belongs_to :provisioning_manager
+
+  scope :ptables, where(:type => "CustomizationScriptPtable")
+  scope :media, where(:type => "CustomizationScriptMedium")
```

Comment:
Err, they should work as methods :confused:

Personally I like scopes for this sort of thing, partly just because it provides a natural grouping, and partly because it back-stops you with a guarantee that you'll end up returning an AR::Relation.

But if they're going to be scopes, they should be using lambdas (forward compatible, and generally more predictable behaviour):

```ruby
scope :media, lambda { where(:type => "CustomizationScriptMedium") }
```